src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java

```diff
@@ -10,42 +10,42 @@
 public class MissingRepositoryException extends SnapshotAPIException {
 
     public static final List<String> GENERIC_REASON = Arrays.asList(
-        "This Overleaf project currently has no git access.",
+        "This Overleaf project currently has no git access, either",
+        "because the project does not exist, or because git access is not enabled for the project.",
```

Comment:
What is the target line length we're going for here? This one is 89. I think probably 72 characters?

Reply:
Yeah, it's mad a bit more tricky by the git client prefixing something like `remote error: something:` on the first line. Will try to balance them out

Reply:
Ah yes, good point. Agree on balancing a bit more, however.